lib/elixir_newbie_web/live/blog_show.ex

```diff
@@ -0,0 +1,62 @@
+defmodule ElixirNewbieWeb.BlogShow do
+  use ElixirNewbieWeb, :live_view
+  alias ElixirNewbie.Blog
+  alias ElixirNewbie.Stats
+  alias ElixirNewbie.Blog.Post
+  ## alias ElixirNewbie.Router.Helpers, as: Routes ##
+
+
+  def render(assigns) do ## BigSpaces Wonders... about embedding the  {/if}  <figure class="makeup-blog"> as in v1
+    ~H"""
+     <section class="bg-[url('/images/BlogBackground.webp')] min-h-screen bg-repeat text-white">
+      <div class="to-black/60 bg-gradient-to-t from-black">
+        <.navigation/>
+
+
+      <section class="mx-auto w-2/3 pb-24">
+        <h1 class="mb-10 block rounded-lg text-4xl hover:opacity-75"><%= raw @blog.title%></h1>
+        <p class="mb-6 text-2xl"><%= raw @blog.description %></p>
+        <p class="mb-6 text-l">By <%= raw @blog.author_name %>. Written on <%= Date.to_string(@blog.date) %> </p>
+        <img class="mb-4 rounded-2xl" src={"/images/posts/#{@blog.cover_image}"}/>
+        <%!-- {#if @blog.livebook_url}
+        <a href={@blog.livebook_url}>
+          <img src="https://livebook.dev/badge/v1/black.svg" alt="Run in Livebook" />
+        </a>
+        {/if} --%>
+        <div class="markdown">
+          <%= raw @blog.body%>
```

Comment:
I'm being nitpicky, but formatting this stuff is important to make sure you are being "clean" otherwise code looks like a messy room with clothing everywhere.

```elixir
<%= raw @blog.body%>
should be
<%= raw @blog.body %>
```

It's missing a space. This might be best handled by a formatter, but as far as I'm aware there isn't great support for formatting HEEx inside of a `render` function.

Reply:
Thanks. Nitpicky is good.